schedule/yast/rmt_server_install/rmt_server_install.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: rmt_server_install
+description: |
+  Perform an installation enabling some modules and selecting some
+  patterns.This test suite always registers to have access to some modules.
+vars:
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_rmt
+  - installation/module_registration/register_nonconflicting_modules
+  - installation/add_on_product/skip_install_addons
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
+  - installation/partitioning
```

Comment:
these 2 modules are needle-based modules, you just need this sequence:
```
   - installation/system_role/accept_selected_role_SLES_with_GNOME 
   - installation/partitioning/accept_proposed_layout
   - installation/clock_and_timezone/accept_timezone_configuration
```

Reply:
Done!